### PR TITLE
Add unit test portfolio create

### PR DESCRIPTION
**Pull request by msm-faraj**
This pr aims to:
add unit test for juse create mothod of the portfolio controller
because I desiced to change controller findAll method of portfolio (add pagination), In this pr I just write the unit test for first method. After this pr I am going to add pagination to portfolio controller and then after that compolete the unit tests.

### CODE REVIEW

File: src/controllers/portfolio/portfolio-controller.spec.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+      findOne: jest.fn(),
+      update: jest.fn(),
+    };
+    mockPortfolioController = new PortfolioController({
+      User: mockUser,
+      Portfolio: mockPortfolio,
+    });
+  });
+
+  describe("instance of class", () => {});
```
Comment (pouyapourbaba) [inline src/controllers/portfolio/portfolio-controller.spec.js:56]: What is the plan here?

File: src/controllers/portfolio/portfolio-controller.spec.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+      it("calls res.status with correct params", () => {
+        expect(mockRes.status).toHaveBeenCalledWith(mockHttpStatusCreated);
+      });
+
+      it("calls res.statu().send with correct params", () => {
+        expect(mockRes.status().send).toHaveBeenCalledWith(mockReq.body);
+      });
+    });
+
+    describe("When something goes wrong", () => {
```
Comment (pouyapourbaba) [inline src/controllers/portfolio/portfolio-controller.spec.js:94]: Add the failure tests for User.findOne and Portfolio.create

File: src/controllers/portfolio/portfolio-controller.spec.js (hunk 1 above)
Comment (msm-faraj) [reply src/controllers/portfolio/portfolio-controller.spec.js:56]: I am going to test instance of this class

File: src/controllers/portfolio/portfolio-controller.spec.js (hunk 2 above)
Comment (msm-faraj) [reply src/controllers/portfolio/portfolio-controller.spec.js:94]: done

**Review by pouyapourbaba — APPROVED** [review]

**Pull request merged**